diff --git a/node_modules/isomorphic-git/index.cjs b/node_modules/isomorphic-git/index.cjs
index b7e8bcd..86e4b55 100644
--- a/node_modules/isomorphic-git/index.cjs
+++ b/node_modules/isomorphic-git/index.cjs
@@ -14092,6 +14092,7 @@ async function statusMatrix({
         }
         const entry = [undefined, headOid, workdirOid, stageOid];
         const result = entry.map(value => entry.indexOf(value));
+        console.log([filepath, headOid, stageOid, workdirOid ]);
         result.shift(); // remove leading undefined entry
         return [filepath, ...result]
       },
