diff --git a/node_modules/isomorphic-git/index.cjs b/node_modules/isomorphic-git/index.cjs
index b7e8bcd..5f45927 100644
--- a/node_modules/isomorphic-git/index.cjs
+++ b/node_modules/isomorphic-git/index.cjs
@@ -918,7 +918,7 @@ function compareStats(entry, stats) {
     e.ctimeSeconds !== s.ctimeSeconds ||
     e.uid !== s.uid ||
     e.gid !== s.gid ||
-    e.ino !== s.ino ||
+    (process.platform !== "win32" && e.ino !== s.ino || e.ino === 0) ||
     e.size !== s.size;
   return staleness
 }
@@ -4201,6 +4201,7 @@ class GitWalkerFs {
       ) {
         const stage = index.entriesMap.get(entry._fullpath);
         const stats = await entry.stat();
+        console.log({stage,stats}) // JRI
         if (!stage || compareStats(stats, stage)) {
           const content = await entry.content();
           if (content === undefined) {
@@ -4232,6 +4233,7 @@ class GitWalkerFs {
       });
       entry._oid = oid;
     }
+    console.log({GitWalkerFsOid: entry._oid}) // JRI
     return entry._oid
   }
 }
@@ -14092,6 +14094,7 @@ async function statusMatrix({
         }
         const entry = [undefined, headOid, workdirOid, stageOid];
         const result = entry.map(value => entry.indexOf(value));
+        console.log([filepath, headOid, stageOid, workdirOid ]); // JRI
         result.shift(); // remove leading undefined entry
         return [filepath, ...result]
       },
