diff --git a/node_modules/isomorphic-git/index.cjs b/node_modules/isomorphic-git/index.cjs
index b7e8bcd..76c8557 100644
--- a/node_modules/isomorphic-git/index.cjs
+++ b/node_modules/isomorphic-git/index.cjs
@@ -4163,6 +4163,9 @@ class GitWalkerFs {
       }
       entry._type = type;
       stat = normalizeStats(stat);
+      if (process.platform === "win32") {
+        stat.ino = 0;
+      }
       entry._mode = stat.mode;
       // workaround for a BrowserFS edge case
       if (stat.size === -1 && entry._actualSize) {
